spring-rabbit-ext/README.adoc

```diff
@@ -0,0 +1,5 @@
+= spring-rabbit-ext
+
+This project provides extensions to `spring-rabbit`.
+
+In particular, the `log4j2 Appender` is contained here, to avoid classpath conflicts with `spring-rabbit`.
```

Comment:
What is the problem with "classpath conflicts"?
Both Log4jS should be `optional` there in the `spring-rabbit` and it is up to end-user to choose an appropriate dep for his requirements.

Please, share your experience and I may agree with you to go ahead with `ext` .
From other side seeing `ext` in the Gradle config I was going to suggest to move other our `Appender`s there as well...